### PR TITLE
Include libyara modules

### PR DESCRIPTION
**Pull request by ikrivosheev**
Add `libyara/modules` for compile libyara

### CODE REVIEW

File: yara-sys/build.rs [hunk 1]
```diff
@@ -3,13 +3,19 @@
 fn main() {
     build::build_and_link();
     bindings::add_bindings();
+    println!("cargo:rustc-link-lib=dylib=ssl");
+    println!("cargo:rustc-link-lib=dylib=crypto");
```
Comment (Hugal31) [inline yara-sys/build.rs:7]: I'm not sure about this.
Also, why do we have to specify this at this place? Shouldn't libyara link the correct libraries?

Comment (ikrivosheev) [reply yara-sys/build.rs:7]: Yara default enable `pe` module witch is depends on https://github.com/VirusTotal/yara/blob/master/libyara/crypto.h which is depends on crypto

Comment (ikrivosheev) [reply yara-sys/build.rs:7]: When link rust-yara with libyara we need this libraries

Comment (ikrivosheev) [reply yara-sys/build.rs]: @Hugal31  I add check `openssl`

**Comment by ikrivosheev** [comment]
@Hugal31 after merge, please, make small fix release

**Comment by Hugal31** [comment]
I've run out of Travis credit and I don't want to merge this PR without testing on Windows. I am looking for a solution.

**Comment by Hugal31** [comment]
Why isn't your Github workflow running? Do I need to approve something?

**Comment by ikrivosheev** [comment]
> Why isn't your Github workflow running? Do I need to approve something?

I don't now why, but I debug in my repository: https://github.com/ikrivosheev/yara-rust/actions

**Pull request closed without merging**